network/p2p/libp2pNode.go

```diff
@@ -514,8 +532,13 @@ func (n *Node) UnSubscribe(topic flownet.Topic) error {
 		return err
 	}
 
+	err := n.pubSub.UnregisterTopicValidator(topic.String())
+	if err != nil {
+		return fmt.Errorf("could not unregister topic validator for (%s): %w", topic, err)
+	}
+
```

Comment:
as you'll see from CI, this fails if there's no validator on the topic (e.g. the subscribe call didn't keep any)